feature-libs/my-account/organization/src/components/user-group/user-group.config.ts

```diff
@@ -0,0 +1,198 @@
+export const userGroupRoutingConfig: RoutingConfig = {
+  routing: {
+    routes: {
+      userGroup: {
+        paths: ['organization/user-groups'],
+      },
+      userGroupCreate: {
+        paths: ['organization/user-groups/create'],
+      },
+      userGroupDetails: {
+        paths: ['organization/user-groups/:code'],
+        paramsMapping: { code: 'uid' },
+      },
+      userGroupEdit: {
+        paths: ['organization/user-groups/:code/edit'],
+        paramsMapping: { code: 'uid' },
+      },
+      userGroupUsers: {
+        paths: ['organization/user-groups/:code/users'],
+        paramsMapping: { code: 'uid' },
+      },
+      userGroupAssignUsers: {
+        paths: ['organization/user-groups/:code/users/assign'],
+        paramsMapping: { code: 'uid' },
+      },
+      userGroupPermissions: {
+        paths: ['organization/user-groups/:code/purchase-limits'],
+        paramsMapping: { code: 'uid' },
+      },
+      userGroupAssignPermissions: {
+        paths: ['organization/user-groups/:code/purchase-limits/assign'],
+        paramsMapping: { code: 'uid' },
+      },
+    },
+  },
+};
+
+export const userGroupCmsConfig: CmsConfig = {
+  cmsComponents: {
+    ManageUserGroupsListComponent: {
+      component: UserGroupListComponent,
+      childRoutes: [
+        {
+          path: 'create',
+          component: UserGroupCreateComponent,
+          canDeactivate: [SplitViewDeactivateGuard],
+        },
+        {
+          path: ':code',
+          component: UserGroupDetailsComponent,
+          canDeactivate: [SplitViewDeactivateGuard],
+          children: [
+            {
+              path: 'edit',
+              component: UserGroupEditComponent,
+              canDeactivate: [SplitViewDeactivateGuard],
+            },
+            {
+              path: 'users',
+              component: UserGroupUserListComponent,
+              canDeactivate: [SplitViewDeactivateGuard],
+              children: [
+                {
+                  path: 'assign',
+                  component: UserGroupAssignUsersComponent,
+                  canDeactivate: [SplitViewDeactivateGuard],
+                },
+              ],
+            },
+            {
+              path: 'purchase-limits',
+              component: UserGroupPermissionListComponent,
+              canDeactivate: [SplitViewDeactivateGuard],
+              children: [
+                {
+                  path: 'assign',
+                  component: UserGroupAssignPermissionsComponent,
+                  canDeactivate: [SplitViewDeactivateGuard],
+                },
+              ],
+            },
+          ],
+        },
+      ],
+      guards: [AuthGuard],
+    },
+  },
+};
+
+export function userGroupTableConfigFactory(): TableConfig {
+  return userGroupTableConfig;
+}
+
+export const userGroupTableConfig: TableConfig = {
+  table: {
+    [OrganizationTableType.USER_GROUP]: [
+      // TODO: consider cascading from smallest size
```

Comment:
This comment is not needed for cost center anymore and shouldn't be multiplicated for other features.